content/docs/start/model-management/model-registry.md

```diff
@@ -16,8 +16,8 @@ management.
 
 <admon type="tip" id="GTO-tip">
```

Comment:
@tibor-mach Does this `id` work? When I tried, the link to this section from the model-cicd page just opens this page and not this admon.

Reply:
It looks like you're right, perhaps probably ids don't work with admon? Since this is at the beginning of the chapter, I did not really notice that it was not redirecting specifically to it.

Reply:
@yathomasi can you confirm this pls? We can then remove the id from here.

Reply:
@tapadipti @tibor-mach we don't link `admonitions`. If we want to link it, we have `<details>` tag which will be linked based on title or custom `id` can also be provided.

eg: https://dvc.org/doc/start/data-management/data-pipelines#stage-expand-to-see-what-happens-under-the-hood

Reply:
Thanks @yathomasi. We have been using `details` in other places, but wanted to confirm if the link in admons works. Since it doesn't I've deleted the `id` and the link to it. @tibor-mach ptal [this commit](https://github.com/iterative/dvc.org/pull/5000/commits/3c82cc2a8818dd0f0f177f3df81aa2744198ddc3) - if it's ok, I'll merge the PR.

Reply:
Thanks Thomas! Sure, this commit works well. It is the beginning of the chapter anyway, so it is enough to link to that.